src/main/java/pl/projekt/cdkeys/CdKeyController.java

```diff
@@ -0,0 +1,64 @@
+package pl.projekt.cdkeys;
+
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.ResponseBody;
+import pl.projekt.author.Author;
+import pl.projekt.author.AuthorDao;
+import pl.projekt.cdkeys.CdKey;
+import pl.projekt.cdkeys.CdKeyDao;
+
+import javax.persistence.TypedQuery;
+import java.util.List;
+import java.util.stream.Collectors;
+
+@Controller
+@RequestMapping("/cdkey")
+public class CdKeyController {
+    private final CdKeyDao cdKeyDao;
+    private final AuthorDao authorDao;
+
+    public CdKeyController(CdKeyDao cdKeyDao, AuthorDao authorDao) {
+        this.cdKeyDao = cdKeyDao;
+        this.authorDao = authorDao;
+    }
+
+    @GetMapping("/create")
+    @ResponseBody
+    public String createBook() {
```

Comment:
book? :)
jeżeli chcesz to pożenić z widokami to nie powinno być response body i rzeczy powinny być odebrne z formularza